admin/pre_reg/templatetags/prereg_extras.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+from django.core.urlresolvers import reverse
+
+register = template.Library()
+
+
+@register.simple_tag
```

Comment:
This would be another good use of that reverse with querystring capability thing we talked about in a different PR. Also,  [furl](https://github.com/gruns/furl/blob/master/API.md#inline-manipulation), which is used throughout the osf codebase would be really useful here vs string interpolation.